demo/src/App.vue

```diff
@@ -25,19 +25,20 @@
         </div>
       </div>
     </div>
+    <div id="fieldset-container">
     <fieldset>
       <legend>Rename a component ID</legend>
       <label>Choose an ID:</label>
       <select
-        v-model="selectedId"
-        name="ids"
-        id="component-id"
+          v-model="selectedId"
```

Comment:
bad indentation, fix it for all tab in template

Reply:
done